pkg/users/account-create-dialog.js

```diff
@@ -118,6 +127,26 @@ function validate_real_name(real_name) {
         return _("The full name must not contain colons.");
 }
 
+function validate_uid(uid, accounts, min_id, max_id, change) {
+    if (!uid)
+        return _("No UID specified");
```

Comment:
This shouldn't be an error IMHO -- then it should just let `useradd` pick the next one.